### DISCUSSION TITLE
Suggest: Geofence: Add costs per 24 hours

### DISCUSSION THREAD

**Discussion by cg089** [teslamate-org/teslamate#1435] [Ideas]
Hi,

there are several parking spots that cost per day (airports, inner cities etc).
After parking there for one day, the cost just appear once.

However, if the car is parked, there are multiple charges added: one full charge and afterwards serveral 1-2-3 minutes charges.

Currently the session costs are then all added, resulting in hundrets of $ per day, when there is effectively just one session.

**Comment by dmachadol** [comment]
And diferent price for all car_id or diferent cost per car_id??
